buf.gen.ts.yaml

```diff
@@ -4,14 +4,20 @@ plugins:
   - local: protoc-gen-es
     strategy: all
     out: ./ts/src/generated/protos
+    include_imports: true
```

Comment:
without this I get runtime error during encoding/decoding of proto messages. This file is missing:
* `google/api/annotations`